README.md

```diff
@@ -170,6 +170,12 @@ Use this command to format the package files.
 
 The formatter supports JSON and YAML format, and skips "ingest_pipeline" directories as it's hard to correctly format Handlebars template files. Formatted files are being overwritten.
 
+### `elastic-package generate`
```

Comment:
"generate" may be a too generic command name, consider using a more specifc name or subcommand.

If the data is intended to be used in benchmars, maybe it could live under the benchmark command, something like `elastic-package benchmark generate-data`.

Reply:
There might be both commands. The `generate` command is focused on generating the sample data sets. I expect there is a benchmark command that can have a flag to generate or use some pregenerated data.

Reply:
> The `generate` command is focused on generating the sample data sets.

But `elastic-package generate` at the root level may sound as if it generates packages. If we want it independent of benchmarking, maybe we can keep it as `elastic-package generate-data` or `elastic-package dataset generate` if there can be more actions on datasets.